broker/bam/src/ba.cc

```diff
@@ -51,7 +51,7 @@ static bool _every_kpi_in_dt(
   }
 
   return true;
-}
+};
```

Comment:
You didn't make a lambda here. You made the same code as the previous one but with the new C++ syntax.
To make a lambda, move this code inside the function using every_kpi_in_dt, it will be better.